_docs/install/kubernetes/index.md

```diff
@@ -31,13 +33,17 @@ of the kubelets. This will be [fixed](https://github.com/kubernetes/kubernetes/p
 For Kubernetes 1.7, you can [install the container directly in
 Docker]({%link _docs/install/docker/index.md %}).
 
+## Installation
+
+You can install StorageOS with Helm or creating Kubernetes spec files.
+
 ## Install with Helm
 
-Firstly, [install Helm](https://docs.helm.sh/using_helm).
+Firstly, [install Helm](https://docs.helm.sh/using_helm). If you are running a rbac enabled cluster, you will need to create a [service account](https://github.com/kubernetes/helm/blob/master/docs/rbac.md).
 
 The StorageOS helm chart triggers an [init container](https://github.com/storageos/init) to enable required kernel configuration on your host. If that procedure doesn't finish successfully, StorageOS container will not start.
 
-If that is your case checkout the [os support](/docs/reference/os_support) page as you can manually prepare the settings for the kernel.
+If that is your case, checkout the [os support](/docs/reference/os_support) page as you can manually prepare the settings for the kernel.
```

Comment:
"If that is your case, checkout" => "If that is the case, check out"

Reply:
Done